lib/scan/wp_timthumbs/engine.py

```diff
@@ -101,7 +102,7 @@ def test(target, retries, timeout_sec, user_agent, http_method, socks_proxy, ver
     if verbose_level > 3:
         info(messages(language, "trying_message").format(trying, total_req, num, total, target_to_host(target), "default_port",
                                                          'wp_timthumb_scan'))
-    if socks_proxy is not None:
+    if socks_proxy != None:
```

Comment:
@aman566 you should not have changed this one: 
According to PEP8 "Comparisons to singletons like None should always be done with is or is not, never the equality operators."
Source: https://www.python.org/dev/peps/pep-0008/